resources/js/bootstrap.js

```diff
@@ -2,7 +2,16 @@ import Vue from 'vue';
 
 import axios from 'axios';
 
-import BootstrapVue from 'bootstrap-vue'
+import BootstrapVue from 'bootstrap-vue';
+
+import VueSweetalert2 from 'vue-sweetalert2';
+
+Vue.use(VueSweetalert2, options);
+
+const options = {
```

Comment:
Shouldn't the options be defined before they are actually used?

Reply:
Yes, sorry, I corrected it.